src/modules/lists/lists.controller.ts

```diff
@@ -70,4 +71,49 @@ export class ListsController {
     };
   }
 
+  @ApiOperation({ title: 'Updates a given list' })
+  @Put('/:listid')
+  async updateList(@Req() request: Request, @Param('userid') userId: string, @Param('listid') listId: string, @Body() data: UpdateList) {
```

Comment:
The `data` parameter should be a type of `ListDto`, this way we can [reuse what we have](https://github.com/Coding-Coach/find-a-mentor-api/blob/master/src/modules/lists/dto/list.dto.ts).